base/applications/charmap/charmap.c

```diff
@@ -567,6 +567,8 @@ PanelOnCreate(HWND hWnd, WPARAM wParam, LPARAM lParam)
         }
     }
 
+    SetFocus(hCharmapDlg);
```

Comment:
Why?

Reply:
This panel of `PanelOnCreate` is not a dialog.
So, it doesn't auto-focus.
`hCharmapDlg` is the first control of the panel.